### PR TITLE
Rename method arguments to use camelCase casing (installation directory)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7467c3f59e3201a2867ec28ab21866cf239d7ad6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31153">issues.joomla.org/tracker/joomla-cms/31153</a>.</sub>

File: installation/model/languages.php [hunk 1]
```diff
@@ -431,16 +431,16 @@ protected function getClient($client = 'administrator')
 	/**
 	 * Set the default language.
 	 *
-	 * @param   string  $language    The language to be set as default.
-	 * @param   string  $cms_client  The name of the CMS client.
+	 * @param   string  $language   The language to be set as default.
```
Comment (zero-24) [inline installation/model/languages.php]:
```suggestion
	 * @param   string  $language    The language to be set as default.
```

To fix drone

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f84599f1cb9ee92fad5f679396076a7a9943dd27<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31153">issues.joomla.org/tracker/joomla-cms/31153</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on f84599f1cb9ee92fad5f679396076a7a9943dd27<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31153">issues.joomla.org/tracker/joomla-cms/31153</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31153">issues.joomla.org/tracker/joomla-cms/31153</a>.</sub>

**Pull request merged**